src/engine/tt_hash_table.h

```diff
@@ -198,12 +216,16 @@ class TtHashTable {
         entry.data.store(newData, std::memory_order_relaxed);
     }
 
+    constexpr static std::size_t tableSizeFromMb(size_t sizeMb)
+    {
+        return (sizeMb * 1024 * 1024) / sizeof(TtHashEntry);
+    }
+
 private:
-    constexpr static std::size_t s_hashTableSizeMb { 128 };
```

Comment:
Why are we changing the default size by a factor of 8?

Reply:
Most engines use 16 as default so it seems to be the better practice. 
Also 128 is actually fairly big ^^